src/components/common/Directors.vue

```diff
@@ -795,7 +797,7 @@ export default class Directors extends Vue {
    * See also LegalServices.fetchParties().
    */
   // FUTURE: this API call should be in the parent component or some mixin/service
-  private async fetchDirectors (): Promise<void> {
+  async fetchDirectors (): Promise<void> {
```

Comment:
There are 3 code smells that are related to the Cognitive Complexity. One of them is here. SonarCloud is complaining about them now after the package updates have been done.

I was wondering, shall I refactor these 3 methods to reduce their cognitive complexity or leave them as if for now? I'm open to suggestions.

Reply:
Leave them for now. I'll tell SC that these aren't issues.

We have to be careful to limit scope in tickets.

Reply:
> Leave them for now. I'll tell SC that these aren't issues.

Done (marked as "won't fix").

Reply:
Awesome, thanks Sev!

Reply:
This is resolved.